metricbeat/module/kibana/stats/stats.go

```diff
@@ -95,8 +94,13 @@ func (m *MetricSet) init() error {
 		return fmt.Errorf(errorMsg, m.FullyQualifiedName(), kibana.StatsAPIAvailableVersion, kibanaVersion)
 	}
 
+	// Add exclude_usage=true if the Kibana Version supports it
+	if kibana.IsUsageExcludable(kibanaVersion) {
+		origURI := statsHTTP.GetURI()
+		statsHTTP.SetURI(origURI + "&exclude_usage=true")
```

Comment:
I'll double check this change. The code in this part is definitely obscure but if the `defer` statement is deleted here, it might be the case that the uri will be `&exclude_usage=true&exclude_usage=true&exclude_usage=true` after 3 periods.

Reply:
it makes perfect sense! Is there any documentation I can use to run this PR? Or any artifacts generated during CI that I can use to test?

Reply:
oh wait you can use https://github.com/elastic/elastic-package to spin up the entire environment and then use `go build` with your code.

Reply:
@sayden I spent a while trying to test it locally, but I'm on an M1 Mac now and there's a library that is no-longer compatible. I'd appreciate some help 😇

Reply:
I have just seen this. I'll do a try asap